core/src/main/java/org/apache/carbondata/core/datamap/TableDataMap.java

```diff
@@ -138,8 +138,12 @@ public CarbonTable getTable() {
       }
     }
     int numOfThreadsForPruning = CarbonProperties.getNumOfThreadsForPruning();
+    int carbonDriverPruningMultiThreadEnableFilesCount =
+        Integer.parseInt(CarbonProperties.getInstance().getProperty(
+            CarbonCommonConstants.CARBON_DRIVER_PRUNING_MULTI_THREAD_ENABLE_FILES_COUNT,
```

Comment:
Need update the document for the new property added

Reply:
resolved